manifests/input.yaml

```diff
@@ -2,452 +2,352 @@ format: yamanifest
-    md5: 425a2b30b996d40e34acd61d319a6faf
+    binhash: fc03d315746c6a0b254a7632dc28295c
+    md5: 78ddfbdc9d9551cf6c6c39fe734716a4
 work/atmosphere/INPUT/cable_vegfunc_N96.anc:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/cable_vegfunc_N96.anc
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/modern/share/atmosphere/land/vegetation/global.N96/2020.05.19/cable_vegfunc_N96.anc
   hashes:
-    binhash: a91053ef6f75a81581f28d0979e4d526
+    binhash: b590c62f20575a65f1f5ee83ec687304
     md5: 7a8cb16191ad29b081514657893530c6
-work/atmosphere/INPUT/ozone_1849_2015_ESM1.anc:
-  fullpath: /g/data/access/payu/access-esm/input/historical/atmosphere/ozone_1849_2015_ESM1.anc
+work/atmosphere/INPUT/def_soil_params.txt:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/land/biogeophysics/resolution_independent/2020.05.19/def_soil_params.txt
+  hashes:
+    binhash: f6bbe84fb16971f00d2ba34eef542976
+    md5: 93107e8c37e04ccf97471fd2ac8f63eb
+work/atmosphere/INPUT/def_veg_params.txt:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/land/biogeophysics/resolution_independent/2020.05.19/def_veg_params.txt
   hashes:
-    binhash: eecbb2a8383e88211e7ca64ee88bed6a
-    md5: 96cded5d4248c88e4088700778dc8557
-work/atmosphere/INPUT/ozone_1850_ESM1.anc:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/ozone_1850_ESM1.anc
+    binhash: 22ce1f400a832868aefeaa9721ac3a48
+    md5: 1f9a27f0e252bf6c466100573c126789
+work/atmosphere/INPUT/modis_phenology_csiro.txt:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/land/biogeochemistry/resolution_independent/2020.05.19/modis_phenology_csiro.txt
   hashes:
-    binhash: 839cf76c5a5784b294824cab29be9e54
-    md5: a64a01e3b092085fca308524a75a1cee
-work/atmosphere/INPUT/qrclim.slt:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/qrclim.slt
+    binhash: a86ffb366359d6d353c6f385789185a1
+    md5: a7b89d25cff82cf13cf812a98ad3a029
+work/atmosphere/INPUT/ozone_1849_2015_ESM1.anc:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/modern/historical/atmosphere/forcing/global.N96/2021.06.22/ozone_1849_2015_ESM1.anc
   hashes:
-    binhash: 3089b0c16e9020719069d301b0d97dab
-    md5: dfb5e52b445917514e5c6ce2ac208c46
-work/atmosphere/INPUT/qrclim.smow:
-  fullpath: /g/data/access/payu/access-esm/input/pre-industrial/atmosphere/qrclim.smow
+    binhash: d0f4f5fa148594a325854548f84b82eb
+    md5: b85f5374d1b89cf23cff05b6c35c4d0f
+work/atmosphere/INPUT/pftlookup_csiro_v16_17tiles_wtlnds.csv:
+  fullpath: /g/data/vk83/configurations/inputs/access-esm1p5/share/atmosphere/land/biogeochemistry/resolution_independent/2020.05.19/pftlookup_csiro_v16_17tiles_wtlnds.csv
   hashes:
-    binhash: 360842c03286f8077514407200ac6f90
-    md5: 89d4912892b8ebd22329dd131d326ccc
+    binhash: 7b29e253f569372abb0a5f2769b58a3d
+    md5: 109d0702afd2cb0a66bf141faccaee30
```

Comment:
`pftlookup_csiro_v16_17tiles_wtlnds.csv` is a different file but isn't listed as one of the known changed files/

Reply:
As above with the same md5 hash `109d0702afd2cb0a66bf141faccaee30` in both cases.